test/migration.js

```diff
@@ -79,7 +79,56 @@ test('migrate moves msgs from old log to new log', (t) => {
   )
 })
 
-test('migrate keeps new log synced with old log being updated', (t) => {
+test('migrate moves msgs from ssb-db to new log', (t) => {
```

Comment:
This test is the same as the previous, but it installs `ssb-db`. This is to make sure that some code paths are covered (maybe someday we should add a code coverage tool).